include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -509,8 +509,7 @@ struct __peer_prefix_helper;
 template <typename _OffsetT>
 struct __peer_prefix_helper<_OffsetT, __peer_prefix_algo::atomic_fetch_or>
 {
-    using _AtomicT = sycl::atomic_ref<::std::uint32_t, sycl::memory_order_relaxed, sycl::memory_scope::sub_group,
-                                      sycl::access::address_space::local_space>;
```

Comment:
Looking into `__atomic_ref` in `__dpl_sycl`,  this seems to change from `sycl::memory_scope::sub_group` to `sycl::memory_scope::work_group`, is this intended, or do we need to add a sub_group memory scope version and use that?

Reply:
In this instance, the work group scope and the sub group scope are the same since the kernel forces the work group size to be the same as the max sub group size of the compute unit. So it shouldn't matter for this change.

I also am not sure if the compiler in general will emit different instructions for a work group scope or a sub group scope. I compared the NVPTX assembly that gets cached in `/tmp` between the two versions of the code and they are identical.

Reply:
Works for me.  I'll review on the other PR.